dart/lib/src/http_client/sentry_http_client.dart

```diff
@@ -101,6 +103,10 @@ class SentryHttpClient extends BaseClient {
       innerClient = BreadcrumbClient(client: innerClient, hub: _hub);
     }
 
+    if (networkTracing) {
+      innerClient = TracingClient(client: innerClient, hub: _hub);
```

Comment:
I believe the TracingClient should be the innermost one. So it should be above the BreadcrumClient. Also if the BreadcrumClient is a child of the TracingClient the current request is also in the breadcrumb list of this trace.

Reply:
makes sense, just moved it, thanks